### ISSUE TITLE
Long queries prepared statements are broken

### ISSUE THREAD

**Issue by prikha** [oelmekki/activerecord_any_of#33]
There is a strange behaviour on queries with more than 9 bind values inside queries, it replaces 

```
$1 => ?
```
However when you reach something like this it corrupts the resulting sql:

```
$12 => ?2
```

Fixed this in #32

**Comment by prikha** [comment]
Guys @oelmekki @shekibobo,please, come take a look. That breaks production stuff using `activerecord_any_of` and pg.

**Comment by oelmekki** [comment]
Hi @prikha , thanks for PR!

I'll review it this week-end, but if you need it urgently, you can
point bundler on your version:

```
gem 'activerecord_any_of', github: 'prikha/activerecord_any_of'
```

**Comment by prikha** [comment]
Thanks @oelmekki  - we`re using the fork - Still would be good if could be merged

**Comment by oelmekki** [comment]
Hi there,

Sorry to have taken so much time. Your PR is now released in 1.4 :)

**Comment by prikha** [comment]
Nice!
